tasks/components/modals/ConfirmDialog.tsx

```diff
@@ -30,7 +32,8 @@ type ConfirmDialogProps = ModalProps & {
   requireAnswer?: boolean,
   onCancel?: () => void,
   onConfirm: () => void,
-  onDecline?: () => void
+  onDecline?: () => void,
+  type: ConfirmDialogType
```

Comment:
I think the name confirmType is more appropriate

Reply:
Excuse me, but I still noticed one small thing. How do you see the  @DasProffi

Reply:
Please use the following instead
```suggestion
  confirmType: ConfirmDialogType
```

Reply:
Hey @florian-sabonchi and @MaxSchaefer. I have pushed a new commit with changes. Please take a look at it.